driver-testsuite/tests/Form/SelectTest.php

```diff
@@ -102,6 +102,28 @@ public function testSetValueMultiSelect()
     }
 
     /**
+     * Set values by Driver
+     */
+    public function testSetValueByOptionText()
```

Comment:
Please rewrite this test to use data providers. This would work like this:

1. data provider defines:
 * select location xpath
 * set values (for `setValue` call)
 * expected got values (from `getValue` call)
2. actual test would accept data provider data from method parameters and do a single assertion.

The change is needed for cases, when 1st assertion fails and it won't be visible that 2nd and 3rd might be passing.